pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -215,6 +218,7 @@ rustPlatform.buildRustPackage rec {
   postFixup = lib.optionalString stdenv.hostPlatform.isLinux ''
     patchelf --add-rpath ${gpu-lib}/lib $out/libexec/*
     patchelf --add-rpath ${wayland}/lib $out/libexec/*
+    wrapProgram $out/libexec/zed-editor --prefix PATH : ${lib.makeBinPath [ nodejs_23 ]}
```

Comment:
this should be --suffix, so that local stuff is picked up over what Nix specifies.